docs/source/process.rst

```diff
@@ -22,75 +21,44 @@ Functionality
 
 - EK80 and EA640 broadband echosounders:
 
-  - calibration based on pulse compression output in the
+  - Calibration based on pulse compression output in the
```

Comment:
Please add `(beta: we found inconsistency between pulse compression outputs from the current echopype implement, EchoView, and Matlab EchoLab code, see #308 )` at the end. Make sure the issue is linked correctly.